tox.ini

```diff
@@ -54,3 +28,14 @@ commands =
     pytest -v {posargs}
 allowlist_externals =
     make
+
+[testenv:black]
+skip_install = true
+deps = pre-commit
+commands = pre-commit run black --all-files
```

Comment:
IMHO no need to keep this section
```suggestion
```

Reply:
I'm totally fine removing it, I only added it because you were concerned about workflows. I'll gladly remove it if you wish.